tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "UMD",
+        "allowJs": true,
+        "checkJs": true,
+        "target": "es6",
+        "moduleResolution": "Node",
+        "noImplicitReturns": true,
+        "jsx": "react",
+        "allowSyntheticDefaultImports": true,
+        "skipLibCheck": true,
+        "baseUrl": ".",
+        "declaration": true,
+        "emitDeclarationOnly": true,
+        "noImplicitAny": true,
+        "strict": false
```

Comment:
Can we add the strict rules that we haven't enabled but leave them commented out? It'd be clearer what they are and what we need to work towards